src/django/api/serializers/activity_log.py

```diff
@@ -0,0 +1,73 @@
+from rest_framework.serializers import (
+    ModelSerializer,
+    SerializerMethodField,
+    EmailField,
+    DateTimeField,
+)
+
+from ..models import User, Submission
+
+
+class ActivityDraftedSerializer(ModelSerializer):
+    user = EmailField(source='created_by.email')
+    action = SerializerMethodField()
+    time = DateTimeField(source='created_at')
+    notes = ''
+
+    def get_action(self, value):
+        return 'Drafted'
+
+    class Meta:
+        model = Submission
+        fields = ['user', 'action', 'time']
+        optional_fields = ['annotations_count', 'notes']
+
+
+class ActivitySubmittedSerializer(ActivityDraftedSerializer):
+    def to_representation(self, instance):
+        obj = super().to_representation(instance)
+        submitter = User.objects.get(pk=instance.submitted_by.pk)
+        obj['action'] = 'Submitted'
+        obj['time'] = instance.submitted_at
+        obj['user'] = submitter.email
```

Comment:
This can be simplified to

```suggestion
        obj['user'] = instance.submitted_by.email
```

Don't need to

```python
submitter = User.objects.get(pk=instance.submitted_by.pk)
```